documentation/users-guide/WritingGUI.md

```diff
@@ -1219,11 +1219,33 @@ place. It is possible to pass a parameter by value to a slot method in
 the following way:
 
 ``` {.cpp}
-Connect(myButton, "Pressed()","TH1",hist, "SetMaximum(=123) ");
-Connect(myButton, "Pressed()","TH1",hist, "Draw(="LEGO")");
+Connect(myButton, "Pressed()", "TH1", hist, "SetMaximum(=123) ");
+Connect(myButton, "Pressed()", "TH1", hist, "Draw(=\"LEGO\")");
 ```
 
 As you see the parameter's value is preceded by the equation symbol (=).
+Please note that this method of passing arguments by value works only
+if the signal has less arguments than the slot.
+If the signal method has the same number of arguments, then the slot
+argument will be overwritten with the signal argument. In the
+example below, the Draw option would be overwritten in the first line
+by the emitted signal argument. The second line would be ok, the
+argument passed by value would not be overwritten.
+
+``` {.cpp}
+Connect(myComboBox, "Selected(const char*)", "TH1", hist, "Draw(=\"LEGO\")");
+Connect(myComboBox, "Selected(const char*)", "TH1", hist, "Draw(const char*, =\"LEGO\")");
```

Comment:
There is no such  `Draw(const char*, const char*)` method in the `TH1` class, could you change this example with another one, either an existing one, or a pseudo class? Thanks!

Reply:
Fixed, I wanted to use TTree::Draw but forgot to change it hehe